src/main/scala/org/allenai/plugins/archetypes/WebappPlugin.scala

```diff
@@ -34,15 +39,18 @@ object WebappPlugin extends AutoPlugin {
     log.info(s"[webapp] NODE_ENV = '$env'")
   }
 
+  val reStartWebappTask = WebappKeys.reStartWebapp := {
+    (NodeKeys.build in Npm).value
+    Revolver.reStart.value
```

Comment:
This differs from the line below which was removed, but I'm assuming works as expected -- just making sure this is correct.